site/docs/es/guide/middleware.md

```diff
@@ -84,7 +84,7 @@ Este stack de funciones es la _middleware stack_.
 (ctx, next) => ...    |
 ```
 
-Volviendo a nuestro ejemplo anterior, ahora sabemos por qué `bot.on(":photo")` ni siquiera se comprobó: el middleware en `bot.on(":text", (ctx) => { ... })` ya se encargó de la actualización, y no llamó a `next`.
```

Comment:
@KnorpelSenf I think here the async is unnecessary (and in the translations). What do you think?

Reply:
I actually agree